yang/frr-vrf.yang

```diff
@@ -0,0 +1,50 @@
+module frr-vrf {
+  yang-version 1.1;
+  namespace "http://frrouting.org/yang/vrf";
+  prefix frr-vrf;
+
+  organization
+    "Free Range Routing";
+  contact
+    "FRR Users List:       <mailto:frog@lists.frrouting.org>
+     FRR Development List: <mailto:dev@lists.frrouting.org>";
+  description
+    "This module defines a model for managing FRR VRF.";
+
+  revision 2019-12-06 {
+    description
+      "Initial revision.";
+  }
+
+  container lib {
+    list vrf {
+      key "name";
+      description
+        "VRF.";
+      leaf name {
+        type string {
+          length "1..36";
+        }
+        description
+          "VRF name.";
+      }
+
+      leaf id {
+        type uint32 {
+          range "1..4294967295";
+        }
+        description
+          "VRF Id.";
+      }
```

Comment:
I think  an additional identifier could inform which vrf backend it is.
a leaf named ns_name optional, which is a string big enough to contain the path of the ns : eg: /var/run/netns/<name>.

Reply:
I have added `ns_name` optional field to check if `netns` feature is enabled.